luzer/luzer_args.c

```diff
@@ -0,0 +1,284 @@
+/*
+ * SPDX-License-Identifier: ISC
+ *
+ * Copyright 2022-2023, Sergey Bronnikov
+ */
+
+#include <lauxlib.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "luzer_args.h"
+#include "macros.h"
+
+#define ENV_NOT_USE_CLI_ARGS "LUZER_NOT_USE_CLI_ARGS"
```

Comment:
I would rename `ENV_NOT_USE_CLI_ARGS` to `IGNORE_CLI_ARGS` and the same for env variable name: `LUZER_NOT_USE_CLI_ARGS` -> `LUZER_IGNORE_CLI_ARGS`.